### PR TITLE
Mark upcoming branch as dev or beta version

### PR DESCRIPTION
**Pull request by dimaqq**
What do you think?
Or maybe `7.0.0b1` or just mark it dynamic for now?

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -8,7 +8,7 @@ build-backend = "setuptools.build_meta"
 [project]
 name = "ops-scenario"
 
-version = "6.1.1"
+version = "7.0.0-beta.1"
```
Comment (tonyandrewmeyer) [inline pyproject.toml:11]: I think the [normal version](https://packaging.python.org/en/latest/specifications/version-specifiers/#pre-releases) would be:

```suggestion
version = "7.0.0beta1"
```

Or even just `7.0beta1`

**Review by tonyandrewmeyer — APPROVED** [review]

File: pyproject.toml [hunk 2]
```diff
@@ -8,7 +8,7 @@ build-backend = "setuptools.build_meta"
...
 
 authors = [
     { name = "Pietro Pasotti", email = "pietro.pasotti@canonical.com" }
```
Comment (PietroPasotti) [inline pyproject.toml:14]: time to add some more people in here @tonyandrewmeyer?

**Review by PietroPasotti — APPROVED** [review]

File: pyproject.toml (hunk 2 above)
Comment (dimaqq) [reply pyproject.toml:14]: And maybe tweak branch protection a bit, so that PR authors can merge approved PRs?

Comment (PietroPasotti) [reply pyproject.toml:14]: added you both as admins

**Pull request merged**